app/renderer/components/config_menu.js

```diff
@@ -39,22 +46,42 @@ module.exports = {
     </div>
     `,
 
-    methods: {
-        clearTasks() {
-            TaskConfig.clearTasks();
-            AppStatus.activeSuite = 0;
-        },
-        resetConfig() {
-            this.config.bottomBar=true;
-            this.config.animatedSpinner=true;
-        },
-        onClose() {
-            this.config.bottomBar = AppStatus.config.bottomBar;
-            this.config.animatedSpinner = AppStatus.config.animatedSpinner;
-        },
-        onSave() {
-            AppStatus.config.bottomBar = this.config.bottomBar;
-            AppStatus.config.animatedSpinner = this.config.animatedSpinner;
-        }
+  methods: {
+    exportTasks(){
+      let content = TaskConfig.getData() ;
+      content = {
+        "suites":content,
+        "version":"0.3.4"
+      };
+      content = JSON.stringify(content) ;
+      dialog.showSaveDialog(
+        {
+          defaultPath: require('os').homedir() + "/gtask.json",
```

Comment:
Few things:
1. using require in the middle of the code is evil, as is hard to spot dependencies, do `const os=require('os')` at the beginning
2. pro node.js tip: You code here is correct, but node provides a built-in module to handle paths, so instead of concatenating strings, using this library you guarantee the path is a valid one (e.g. backslash in windows):
```
const path=require('path');
const os=requie('os');
// rest of the code
defaultPath: path.join(os.homedir(),"gtask.json")